python/alibiexplainer/alibiexplainer/__main__.py

```diff
@@ -16,25 +16,47 @@
 import argparse
 from alibiexplainer import AlibiExplainer
 from alibiexplainer.explainer import ExplainerMethod #pylint:disable=no-name-in-module
+from typing import Dict
+import dill
+import os
+import json
 
-DEFAULT_MODEL_NAME = "model"
+DEFAULT_EXPLAINER_NAME = "explainer"
+EXPLAINER_FILENAME = "explainer.dill"
+CONFIG_ENV = "ALIBI_CONFIGURATION"
+
+ENV_EXPLAINER_URI = "EXPLAINER_URI"
```

Comment:
IMO, this should be STORAGE_URI. We've discussed all components in the graph can discover the mounted model dir at this URI. @rakelkar

Reply:
The only caveat I would see is it we want whitebox explainers they might need both a URi for a saved trained explainer and a uri for the model they need to load.

Reply:
I didn't realize there was a usecase for both. Can we make that transparent? Any asset that an explainer might need simply needs to be provided at StorageURI?

Reply:
I suppose yes they could be together in one bucket. There might need to be a naming convention for what's to be found in the bucket but that is ok and I like have 1 way to specify this across all types. I'll update.

Reply:
+1 on this. Use single bucket for the artifacts